base/src/functions/text.rs

```diff
@@ -342,6 +342,50 @@ impl Model {
         CalcResult::new_args_number_error(cell)
     }
 
+    pub(crate) fn fn_unicode(&mut self, args: &[Node], cell: CellReferenceIndex) -> CalcResult {
+        if args.len() == 1 {
+            let s = match self.evaluate_node_in_context(&args[0], cell) {
+                CalcResult::Number(v) => format!("{}", v),
+                CalcResult::String(v) => v,
+                CalcResult::Boolean(b) => {
+                    if b {
+                        "TRUE".to_string()
+                    } else {
+                        "FALSE".to_string()
+                    }
+                }
+                error @ CalcResult::Error { .. } => return error,
+                CalcResult::Range { .. } => {
+                    // Implicit Intersection not implemented
+                    return CalcResult::Error {
+                        error: Error::NIMPL,
+                        origin: cell,
+                        message: "Implicit Intersection not implemented".to_string(),
+                    };
+                }
+                CalcResult::EmptyCell | CalcResult::EmptyArg => {
+                    return CalcResult::Error {
+                        error: Error::VALUE,
+                        origin: cell,
+                        message: "Empty cell".to_string(),
+                    }
+                }
+            };
+
+            if s.len() == 0 {
+                return CalcResult::Error {
+                    error: Error::VALUE,
+                    origin: cell,
+                    message: "Empty cell".to_string(),
+                };
+            }
+
+            let unicode_number = s.chars().nth(0).unwrap() as u32;
```

Comment:
Sorry, I missed this. We can't use `unwrap` (or `panic`) in the library code.

Reply:
Maybe something like:

```Rust
match s.chars().next() {
    Some(c) => {
        return CalcResult::Number(c as f64);
    },
    None => {
        return CalcResult::Error {
            error: Error::VALUE,
            origin: cell,
            message: "Empty cell".to_string(),
        };
    }
}
```

Reply:
Updated! Can't cast `char` directly to `f64`, has to go through `u32` first